pyfar/classes/coordinates.py

```diff
@@ -1590,14 +1590,18 @@ def find_within(
             ``'spherical_meter'``
                 distance is determined by the great-circle distance
                 expressed in meters.
-        atol : float
-            Absolute tolerance for distance. The default is 1e-15.
-        radius_tol : float
-            Maximum allowed difference within radii for spherical distance
-            measures. The default is 1e-15.
+        atol : float, None
+            Absolute tolerance for distance. The default is 2*eps, where eps
+            is the resolution of the data type of the coordinates.
```

Comment:
If were not using eps, I would suggest to be a bit more clear in the docsring.
```suggestion
            Absolute tolerance for distance. The default ``None`` uses a tolerance of two times the decimal resolution, which is determined from the data type of the coordinate points using ``numpy.finfo``.
```